stream-chat-android-client/src/main/java/io/getstream/chat/android/client/socket/ChatSocketService.kt

```diff
@@ -22,8 +22,8 @@ import io.getstream.chat.android.client.events.ConnectedEvent
 import io.getstream.chat.android.client.models.User
 
 internal interface ChatSocketService {
```

Comment:
What about renaming it to `ChatSocket`??
The same with its implementation class.
On the other hand, do we really need this interface?
It is an internal interface and we only have one implementation (The one we have on tests-side could extend the class and override the same methods without needing an interface

Reply:
I also thought of removing the interface for the same reason. I will remove it.